src/fluree/db/datatype.cljc

```diff
@@ -50,55 +90,306 @@
     (number? x) const/$xsd:decimal
     (boolean? x) const/$xsd:boolean))
 
-(defn coerced
+
+#?(:cljs
+   (defn- left-pad
```

Comment:
why did you implement this fn yourself when you could have just used another [dependency](https://www.theregister.com/2016/03/23/npm_left_pad_chaos/)?

😄

Reply:
I was definitely chuckling to myself when I realized I needed `left-pad` here...